content/docs/command-reference/exp/show.md

```diff
@@ -71,18 +71,24 @@ metric or param.
   paginator.
 
 - `--include-params <list>` - show the specified `dvc params` in the table only.
-  Accepts a comma-separated `list` of param names (including groups).
+  Accepts a comma-separated `list` of param names (including groups). Names can
+  be defined using
+  [wildcard patterns](https://docs.python.org/3.6/library/fnmatch.html)
```

Comment:
Let´s go for 

``Shell style wildcards supported: `*`, `?`, `[seq]`, `[!seq]`, and `**` ``

Just like in the `add --glob` [ref](https://dvc.org/doc/command-reference/add#options).